communication.py

```diff
@@ -128,25 +164,24 @@ def pckParser(self, line):
         longitude_car = line[65] + line[66] + line[67] + line[68]
         [longitude_car] = struct.unpack("f", longitude_car)
 
-        status = line[69]
-        status = int.from_bytes(status, "little", signed=False)
-        yaw = line[70]+line[71]+line[72]+line[73]
-        [yaw] = struct.unpack("f", yaw)
-        roll = line[74]+line[75]+line[76]+line[77]
-        [roll] = struct.unpack("f", roll)
-        pitch = line[78]+line[79]+line[80]+line[81]
-        [pitch] = struct.unpack("f", pitch)
+        """
+
+        """
```

Comment:
remove all old lines. these are unusable for this year. delete all, not comment.